README.md

```diff
@@ -103,3 +103,13 @@ CREATE TRIGGER notify_event
 ```
 
 This will emit an MQTT event with the topic as your event type for all your contract's events.
+
+## Terraform
+
+You can deploy this repo on GCP using Terraform.
+
+To do so, you first need to create a GCP project.
+
+Once done, run `PROJECT_ID=<YOUR_PROJECT_ID> terraform/init.sh` to enable the required Google APIs, create a service account, and download the credentials file.
```

Comment:
I think we might want to include here something about requiring that the user's machine includes specific commands

For example inside some of the new content there are references to `cloud-sql-proxy` (which needs to not only be installed but also granted access), and `jq` (not sure if we can expect this to be on every machine)

I bring this up because anytime you use scripting in Terraform, especially with provisioners, you can end up with corrupted state if the script can't run

How about?

- [ ] Add here explanation of which dependencies you need
- [ ] Mention how you'll need to have cloud-sql-proxy authenticated, then maybe link to the docs (Econia v4 has a link that you can probably use)
- [ ] Add a script that verifies you have what you need, and instructions here on how to run it (e.g. it errors out if you don't have `jq`)

Reply:
What do you mean by

> have cloud-sql-proxy authenticated

?

Reply:
So unfortunately just having `cloud-sql-proxy` in your path isn't enough: you have to setup something called "application default credentials": https://github.com/econia-labs/econia/tree/main/src/terraform/internal-dss#connect-to-cloud-sql. If there were no authentication procedure, then anyone could just connect to anyone's DB

I recall that this authentication step is in addition to what the `glcloud` CLI does by default, and if so then is necessary for folks to run the migrations

- [ ] Incorporate directions from https://github.com/econia-labs/econia/tree/main/src/terraform/internal-dss#connect-to-cloud-sql in README or init script as appropriate

Reply:
I don't think this is needed since the run migrations script uses the credentials from the JSON file downloaded by the init script.

Reply:
Got it, thanks for clarifying